broadcastclient/broadcastclient.go

```diff
@@ -164,7 +173,13 @@ func (bc *BroadcastClient) Start(ctxIn context.Context) {
 				break
 			}
 			log.Warn("failed connect to sequencer broadcast, waiting and retrying", "url", bc.websocketUrl, "err", err)
-			timer := time.NewTimer(5 * time.Second)
+			if sourcesConnectedGauge.Value() <= 0 {
+				log.Error("no connected feed servers")
```

Comment:
IIRC we can't use metrics for stuff like this because if metrics aren't enabled it doesn't work, right?

Reply:
:+1: